fix(human-readable): instantiation is excessively deep (#170)
wagmi-dev/abitype: .changeset/shiny-houses-poke.md

---
"abitype": patch
---

> Fixes type instantiation is excessively deep and possibly infinite bug on `formatAbiParameter` with `exactOptionalPropertyTypes` set to false.

Reviewer: ```suggestion
Fixed bug where type instantiation is excessively deep and possibly infinite when using `formatAbiParameter` with `exactOptionalPropertyTypes` set to `false`.
```
